subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/RepositoryHandlerExtensions.kt

```diff
@@ -90,3 +90,10 @@ fun RepositoryHandler.ivy(url: Any, action: IvyArtifactRepository.() -> Unit) =
         it.setUrl(url)
         it.action()
     }
+
+
+/**
+ * Adds the Kotlin Dev bintray repository.
+ */
+fun RepositoryHandler.kotlinDev() =
```

Comment:
Since it is a public method. Does it make sense to contribute that upstream to gradle/gradle? 🤔